api/src/main/java/org/apache/iceberg/io/CloseableIterable.java

```diff
@@ -75,6 +76,34 @@ public CloseableIterator<E> iterator() {
     };
   }
 
+  /**
+   * Will run the given runnable when {@link CloseableIterable#close()} has been called.
+   *
+   * @param iterable             The underlying {@link CloseableIterable} to iterate over
+   * @param onCompletionRunnable The runnable to run after the underlying iterable was closed
+   * @param <E>                  The type of der underlying iterable
```

Comment:
"der" -> "the"

Reply:
I accidentally leaked german into the code, good catch :)